iscan/static/iscan/query-detail/query_detail.html

```diff
@@ -1,18 +1,18 @@
 <div class="row" style="height: 93%;">
     <div class="col-md-9" style="height:100%">
         <div class="annotation-plot">
-            <annotation-plot data="utterance" selected-annotation="selectedAnnotation"></annotation-plot>
+            <annotation-plot data="utterance" selected-annotation="selectedAnnotation" can_edit="can_edit"></annotation-plot>
         </div>
 
         <div class="detail-plot">
             <waveform-plot data="waveform" begin="utterance.begin" end="utterance.end"
-                           selected-annotation="selectedAnnotation"></waveform-plot>
+                           selected-annotation="selectedAnnotation" can-edit="can_edit"></waveform-plot>
```

Comment:
`can_edit` should be `can_edit`?

Reply:
No, it's to do with how directives handle the variables being moved into scope, I couldn't get can_edit to work as a name, only can-edit -> canEdit